src/engine/renderer/tr_init.cpp

```diff
@@ -875,7 +878,7 @@ ScreenshotCmd screenshotPNGRegistration("screenshotPNG", ssFormat_t::SSF_PNG, "p
 
 		glState.stackIndex = 0;
 
-		for ( i = 0; i < MAX_GLSTACK; i++ )
+		for ( int i = 0; i < MAX_GLSTACK; i++ )
```

Comment:
Bikeshedding, but you can use `size_t` here.

I also wonder if `glConfig2.maxTextureUnits` can be but doing that may be too much intrusive as it is compared with `int` from legacy parts of the code. This can be done later, I'm just wondering.

Reply:
Probably, yeah. That part was just me moving the variable declaration into the loop, but it doesn't look like those numbers will get anywhere close to overflowing: maximum texture units have been at 192 for years and MAX_GLSTACK is just defined as 5, and it doesn't look like we need it to be higher.

Reply:
Yeah, it's not blocking, just a nitpick remark without making anything required. The PR is fine as is.